docs/time/index.rst

```diff
@@ -928,6 +928,26 @@ the TDB timescale::
 
     >>> time_barycentre = times.tdb + ltt_bary
 
+By default, the light travel time is calculated using the position and velocity
+of Earth and the Sun from built-in `ERFA <https://github.com/liberfa/erfa>`_ routines,
+but one can also use more precise calculations using the JPL ephemerides (which are derived from
+dynamical models). An example using the JPL ephemerides is:
+
+.. doctest-requires:: jplephem
+
+    >>> ltt_bary_jpl = times.light_travel_time(ip_peg, ephemeris='jpl') # doctest: +REMOTE_DATA +IGNORE_OUTPUT
+
+The difference between the builtin ephemerides and the JPL ephemerides is normally
```

Comment:
to the precision that's output by default, they look identical to the barycentric corrections with the default ephemeris, so I wonder if it will just look confusing.